token-erc-20/chaincode-go/chaincode/token_contract.go

```diff
@@ -84,11 +83,104 @@ func (s *SmartContract) Mint(ctx contractapi.TransactionContextInterface, amount
 	totalSupply += amount
 	ctx.GetStub().PutState(totalSupplyKey, []byte(strconv.Itoa(totalSupply)))
 
+	// Emit the Transfer event
+	transferEvent := map[string]string{"from": "0x0", "to": minter, "value": strconv.Itoa(amount)}
+	transferEventJSON, err := json.Marshal(transferEvent)
+	if err != nil {
+		return fmt.Errorf("failed to obtain JSON encoding: %v", err)
+	}
+	err = ctx.GetStub().SetEvent("Transfer", transferEventJSON)
+	if err != nil {
+		return fmt.Errorf("event failed to register: %v", err)
```

Comment:
Register is not the correct word here (and elsewhere).

```suggestion
		return fmt.Errorf("failed to set event: %v", err)
```